js/render/draw_background.js

```diff
@@ -3,7 +3,8 @@
 var TilePyramid = require('../source/tile_pyramid');
 var pyramid = new TilePyramid({ tileSize: 512 });
 var util = require('../util/util');
-var EXTENT = require('../data/bucket').EXTENT;
+var Tile = require('../source/tile');
+var pixelsToTileUnits = new Tile().pixelsToTileUnits;
```

Comment:
@lucaswoj `pixelsToTileUnits` is used in `draw_fill` `draw_line` `draw_symbol` and `painter` as well. should I split this refactor into another PR or keep it in this one?